### PR TITLE
[77] implemented creating and deleting of a project

### PR DESCRIPTION
**Pull request by moklidia**

### CODE REVIEW

File: lib/uffizzi/cli/project.rb [hunk 1]
```diff
@@ -62,5 +113,22 @@ def print_projects(projects)
     def set_default_project(project)
       ConfigFile.write_option(:project, project[:slug])
     end
+
+    def validate_options(options)
+      raise Uffizzi::Error.new('Name must be present') if options[:name].nil?
```
Comment (the38th) [inline lib/uffizzi/cli/project.rb:118]: how `name` option could be nil if it's marked as required at line 25?

File: lib/uffizzi/cli/project.rb [hunk 2]
```diff
@@ -62,5 +113,22 @@ def print_projects(projects)
...
+      raise Uffizzi::Error.new('Slug must be present') if options[:slug].nil?
+
+      length_limit = 256
+      name_too_long = options[:name].length > length_limit
+      raise Uffizzi::Error.new("Name must be less than #{length_limit} characters long") if name_too_long
+
+      description_too_long = options[:description] && options[:description].length > length_limit
+      raise Uffizzi::Error.new("Description must be less than #{length_limit} characters long") if description_too_long
+
+      return unless options[:slug]
```
Comment (the38th) [inline lib/uffizzi/cli/project.rb:128]: can this condition be fulfilled after condition on line 119?

File: lib/uffizzi/cli.rb [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'uffizzi'
+require 'byebug'
```
Comment (7R41N33) [inline lib/uffizzi/cli.rb:4]: It's not necessary

File: lib/uffizzi/cli/project.rb [hunk 4]
```diff
@@ -62,5 +113,22 @@ def print_projects(projects)
     def set_default_project(project)
       ConfigFile.write_option(:project, project[:slug])
     end
+
+    def validate_options(options)
+      raise Uffizzi::Error.new('Name must be present') if options[:name].nil?
+      raise Uffizzi::Error.new('Slug must be present') if options[:slug].nil?
+
+      length_limit = 256
```
Comment (7R41N33) [inline lib/uffizzi/cli/project.rb:121]: Let's move this into the configuration file

File: test/uffizzi/helpers/project_helper.rb [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Uffizzi
```
Comment (7R41N33) [inline test/uffizzi/helpers/project_helper.rb:3]: Why did we create this module in `test` folder?

**Pull request merged**